lemur/notifications/cli.py

```diff
@@ -17,22 +18,26 @@
 from lemur.notifications.messaging import send_authority_expiration_notifications
 from lemur.notifications.messaging import send_security_expiration_summary
 
-manager = Manager(usage="Handles notification related tasks.")
+
+@click.group(name="notifications", help="Handles notification related tasks.")
```

Comment:
```suggestion
@click.group(name="notify", help="Handles notification related tasks.")
```
Should the click group name and registered command name be the same? If so, we'd want `notify` here